README.md

```diff
@@ -0,0 +1,72 @@
+# elasticache
+
+Provides an ElastiCache Replication Group resource.
+
+Example Usage
+----------------
+
+```hcl
+module "elasticache" {
+  source = "git@github.com:techservicesillinois/terraform-aws-elasticache"
+
+  cluster_name       = "service-name"
+
+  network_configuration {
+    vpc  = "name-of-vpc"
+    tier = "public"
+    security_group_names = "service-redis-clients"
+  }
+}
+```
+
+Argument Reference
+-----------------
+
+The following arguments are supported:
+
+* `cluster_name` - (Required) The name or identifier of the Replication group.  
+
+* `security_group_ids` - (Required) One or more Amazon VPC security groups associated with this replication group. Use this parameter only when you are creating a replication group in an Amazon Virtual Private Cloud
+
+* `redis_clusters` - (Required) The number of cache clusters (primary and replicas) this replication group will have. If Multi-AZ is enabled, the value of this parameter must be at least 2. Updates will occur before other modifications. Defaults to `2`.
+
+* `redis_node_type` - (Required) The compute and memory capacity of the nodes in the node group. Defaults to `cache.t2.small`.
+
+* `redis_failover` - (Optional) Specifies whether a read-only replica will be automatically promoted to read/write primary if the existing primary fails. If true, Multi-AZ is enabled for this replication group. If false, Multi-AZ is disabled for this replication group. Must be enabled for Redis (cluster mode enabled) replication groups. Defaults to `true`.
+
+* `redis_version` - (Optional) The version number of the cache engine to be used for the cache clusters in this replication group. Defaults to `3.2.10`.
```

Comment:
Should we default to a newer version of the Elasticache redis engine? The latest version supported is Redis 5.0.4 (see https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/supported-engine-versions.html). I have nothing against 3.2.10, but it would seem that we shouldn't start out a new service using an old version of a key component.